SpecFlowMarsProject/StepDefinitions/LanguagePageStepDefinitions.cs

```diff
@@ -0,0 +1,104 @@
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using SpecFlowMarsProject.Pages;
+using SpecFlowMarsProject.Tests;
+using SpecFlowMarsProject.Utilities;
+using System;
+using System.Diagnostics;
+using TechTalk.SpecFlow;
+
+namespace SpecFlowMarsProject.StepDefinitions
+{
+    [Binding]
+    public class LanguagePageStepDefinitions : CommonDriver
+    {
+        //IWebDriver driver;
+        //IWebDriver driver = new ChromeDriver();
+        LoginPage loginPageObj = new LoginPage();
```

Comment:
Please make use of constructors, please see 
 https://www.tutorialspoint.com/csharp/csharp_classes.htm

https://www.knowledgehut.com/tutorials/csharp/csharp-constructors